pkgs/development/python-modules/python3-gnutls/default.nix

```
@@ -0,0 +1,50 @@
+{ lib, fetchFromGitHub, buildPythonPackage, isPy3k, gnutls, twisted, pyopenssl
+, service-identity }:
+
+buildPythonPackage rec {
+  pname = "python3-gnutls";
+  version = "3.1.9";
+
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "AGProjects";
+    repo = "python3-gnutls";
+    rev = "324b78f7cd3d9fe58c89c7f0b2bf94199bd6a6e5"; # version not tagged
+    sha256 = "sha256-18T8bAHlNERHobsspUFvSC6ulN55nrFFb5aqNwU8T00=";
+  };
+
+  propagatedBuildInputs = [ twisted pyopenssl service-identity ];
+
+  patches = [ ./libgnutls-path.patch ];
+  postPatch = ''
+    sed -i 's|@gnutlslib@|'"${lib.getLib gnutls}/lib"'|g' gnutls/library/__init__.py
+  '';
```

There's a common helper for this
```suggestion
  patches = [ 
    (substituteAll {
      src = ./libgnutls-path.patch;
      gnutlslib = "${lib.getLib gnutls}/lib";
    })
   ];
```

---

Ok, thanks. Changed.